File: week-07/day-01/cuboid.js
'use strict';

// Write a program that stores 3 sides of a cuboid as variables (floats)
// The program should write the surface area and volume of the cuboid like:
//
// Surface Area: 600
// Volume: 1000

var a = 10;
var b= 20;
var c= 30;

var surface;
var volume;

volume = a * b* c;
surface = 2*(a*b+b*c+c*a);

console.log("Volume: " + volume);
console.log("Surface: " + surface);

File: week-02/day-01/cuboid.py
# Write a program that stores 3 sides of a cuboid as variables (doubles)
# The program should write the surface area and volume of the cuboid like:
# Surface Area: 600
# Volume: 1000

a = 10
b = 40
c = 60

volume = a * b * c
surface = 2 * ((a * b) + (c * a) + (c * b))
print(volume)
print(surface)
